### DISCUSSION TITLE
v17.4.2

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#954] [General]
## Releases Notes for 17.4.2
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v17.4.2/draw.io-17.4.2-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v17.4.2/draw.io-17.4.2-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v17.4.2/draw.io-universal-17.4.2.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v17.4.2/drawio-amd64-17.4.2.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v17.4.2/drawio-x86_64-17.4.2.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v17.4.2/drawio-x86_64-17.4.2.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Updates electron 18
- Updates to [draw.io core 17.4.2](https://github.com/jgraph/drawio/blob/v17.4.2/ChangeLog). All changes from 17.2.5 to 17.4.2, inclusive, are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v17.4.2'>v17.4.2</a>.</em>

**Comment by kiravi** [comment]
Broken Windows 10 installer. Does not launch. Event after a clean uninstall/re-install.

**Reply by dima60216** [reply]
Windows 10 Pro 10.0.19042 - all good

**Reply by kiravi** [reply]
The latest 21H2 build is: 19044.1645
The installation is smooth. On launch, crashes i.e. does not keep the interface open on clicking create new.

**Reply by kiravi** [reply]
Was able to execute only when "Run As Adminstrator"

**Reply by michaelebrennan** [reply]
agreed on the Win10 install, crashes after removing an item on existing file, or starting an empty file.  Run-As Admin, will attempt.

**Reply by michaelebrennan** [reply]
NO GO. crashes after 1 item modified. 'Run As Administrator' did not help.  Install looked good, won't keep the .io files open.

**Reply by michaelebrennan** [reply]
17.2.4, returning to stable release.

**Reply by 71walceli** [reply]
Just rolled back to previous release!. I even went so far as to rename `C:\Users\Dell\AppData\Roaming\draw.io`, but didn't help!
